source/_integrations/zha.markdown

```diff
@@ -98,13 +102,13 @@ zha:
 
 If you are use ZiGate, you have to use some special usb_path configuration:
 
-- ZiGate USB TTL or DIN: `/dev/ttyUSB0` or `auto` to auto discover the zigate
-- PiZigate : `pizigate:/dev/serial0`
-- Wifi Zigate : `socket://[IP]:[PORT]` for example `socket://192.168.1.10:9999`
+- ZiGate USB-TTL or USB-DIN: `/dev/ttyUSB0` or `auto` to auto discover the zigate
+- PiZigate: `pizigate:/dev/serial0`
+- ZiGate Pack WiFi: `socket://[IP]:[PORT]` for example `socket://192.168.1.10:9999`
 
 {% configuration %}
 radio_type:
-  description: One of `deconz`, `ezsp`, `ti_cc`, `xbee` or `zigate`.
+  description: One of `ezsp`, `xbee`, `deconz`, `zigate`, or `ti_cc`.
```

Comment:
Not sure of the reasoning behind this change?
It was nicely sorted alphabetically, now it seems random?